### ISSUE TITLE
Missing face_detection_short_range.tflite file

### ISSUE THREAD

**Issue by iamlun** [google/mediapipe#4356]
### OS Platform and Distribution

Ubuntu 22.04 (wsl)

### Programming Language and version

java

### Android AAR (if android)

Yes

### Describe the problem

```shell
Follow the document:https://developers.google.com/mediapipe/framework/getting_started/android_archive_library

to run "bazel build -c opt mediapipe/graphs/face_detection:face_detection_mobile_gpu_binary_graph"
to get face_detection_short_range.tflite.
BUT did'n got this file, only got .binarypb file.
```

### Complete Logs

```shell
After build:

WARNING: /home/lun/mediapipe/mediapipe/framework/BUILD:54:24: in cc_library rule //mediapipe/framework:calculator_cc_proto: target '//mediapipe/framework:calculator_cc_proto' depends on deprecated target '@com_google_protobuf//:cc_wkt_protos': Only for backward compatibility. Do not use.
WARNING: /home/lun/mediapipe/mediapipe/framework/tool/BUILD:197:24: in cc_library rule //mediapipe/framework/tool:field_data_cc_proto: target '//mediapipe/framework/tool:field_data_cc_proto' depends on deprecated target '@com_google_protobuf//:cc_wkt_protos': Only for backward compatibility. Do not use.
INFO: Analyzed target //mediapipe/graphs/face_detection:face_detection_mobile_gpu_binary_graph (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //mediapipe/graphs/face_detection:face_detection_mobile_gpu_binary_graph up-to-date:
  bazel-bin/mediapipe/graphs/face_detection/face_detection_mobile_gpu.binarypb
INFO: Elapsed time: 3.190s, Critical Path: 0.02s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action

when I try to copy the file:
cp: cannot stat 'mediapipe/modules/face_detection/face_detection_short_range.tflite': No such file or directory
```

**Comment by ayushgdev** [comment]
Hello @iamlun The documentation does not mention that the `bazel build..` command will generate the TfLite file. It will generate the `.binarypb file`. But the TFLite model is an asset so it has to be downloaded manually and copied to the directory with `cp` command. 
This confusion gives us a hint that we should probably rephrase that section of the document. Will do that.
For now, you can download the TfLite model from the [model cards page](https://github.com/google/mediapipe/blob/master/docs/solutions/models.md#face-detection).

**Comment by iamlun** [comment]
Got it! Thanks. :)

**Comment by jeremmoore** [comment]
@ayushgdev  hi, in the [model cards page](https://github.com/google/mediapipe/blob/master/docs/solutions/models.md#face-detection), the face mesh models(with or without attention) all output 468 landmarks. But for the face blendshape solution, the face mesh v2(supporting 478 landmarks) is needed, I wonder how can I download the face mesh v2 model.

**Comment by abdee67** [comment]
> hello , i am trying to download the mediapipe tflite model face_detection_short_range.tflite but i am unable to find the model on the website/GitHub can anyone help, its urgent

did you get the model man?

**Comment by harshwanjari21** [comment]
> > hello , i am trying to download the mediapipe tflite model face_detection_short_range.tflite but i am unable to find the model on the website/GitHub can anyone help, its urgent
> 
> did you get the model man?

Yes, I was able to get the model. If you install MediaPipe in your IDE, the face_detection_short_range.tflite and face_detection_full_range.tflite models are already included as part of the package. I wrote a small script to locate where Python libraries are installed on my system, and then used another script to extract the model file from the MediaPipe directory. That way, I was able to access and use the model directly.
